poem.py

```diff
@@ -7,11 +7,12 @@ def make_poem(words):
         f"A {random.choice(words)} whispers, passing by.",
         f"In the night a {random.choice(words)} glows,",
         f"And the {random.choice(words)} quietly grows.",
-        f"While {random.choice(words)} echoes in prose."
+        f"While {random.choice(words)} echoes in prose.",
     ]
     return "\n".join(lines)
 
 
 if __name__ == "__main__":
-    words = ["tree", "river", "dream", "song"]
+    raw = input("Enter words (comma-separated): ")
```

Comment:
Thanks! One small robustness tweak before approving:

- If the user just presses Enter (empty input), `words` becomes empty and `make_poem` will crash on `random.choice`.
- Can we fallback to defaults when no words are provided?

Suggested acceptance criteria:
- Empty input → print a friendly message and use defaults.